src/Fable.Arch/Fable.Arch.Html.fs

```diff
@@ -205,6 +205,15 @@ module Attributes =
             |> String.concat " "
             |> Class
 
+    /// Helper to build space separated class with a static part
+    let classBaseList b (list: (string*bool) seq) =
+        list
+            |> Seq.filter (fun (c,cond) -> cond)
+            |> Seq.map (fun (c, cond) -> c)
+            |> String.concat " "
+            |> sprintf "%s %s" b
+            |> Class
```

Comment:
not sure I like the capital `C` in `Class`, but I guess lowercase is not allowed. Maybe that function should be renamed to `className` or something else?